<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSettingsDialog</class>
 <widget class="QDialog" name="PlotSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/application_edit.png</normaloff>:/icons/icons/application_edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General Settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="title"/>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="show_legend">
          <property name="text">
           <string>Show Legend</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>X Axis</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Label</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="xlabel"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="use_xmin">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="xmin"/>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="use_xmax">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="xmax"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Y Axis</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Label</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ylabel"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="use_ymin">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="ymin"/>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="use_ymax">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="ymax"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Job-Query</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Note: If left empty, global job query is used.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="query"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="test">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>SELECTing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="n_results">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>jobs</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>SQL Errors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="query_results">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>102</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>show_legend</tabstop>
  <tabstop>xlabel</tabstop>
  <tabstop>use_xmin</tabstop>
  <tabstop>xmin</tabstop>
  <tabstop>use_xmax</tabstop>
  <tabstop>xmax</tabstop>
  <tabstop>ylabel</tabstop>
  <tabstop>use_ymin</tabstop>
  <tabstop>ymin</tabstop>
  <tabstop>use_ymax</tabstop>
  <tabstop>ymax</tabstop>
  <tabstop>query</tabstop>
  <tabstop>test</tabstop>
  <tabstop>query_results</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
